From 7af7c3ef6305fddd508acf8be6bff5e6926d6b60 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Thu, 17 Dec 2020 17:26:01 +0530
Subject: [PATCH] [REVERT ME] Disable kernel compatibility check

As lld, lto and cfi is disabled due to suspend/resume issue,
kernel compatibility check shows error manufaturer dialog.

Disable kernel compatibility check until suspend/resume fix
is addressed and lld, lto, cfi are enabled.

Tracked-On: OAM-95489
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 RuntimeInfo.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/RuntimeInfo.cpp b/RuntimeInfo.cpp
index df09958..680c631 100644
--- a/RuntimeInfo.cpp
+++ b/RuntimeInfo.cpp
@@ -89,13 +89,14 @@ bool RuntimeInfo::checkCompatibility(const CompatibilityMatrix& mat, std::string
 
     // mat.mSepolicy.sepolicyVersion() is checked against static
     // HalManifest.device.mSepolicyVersion in HalManifest::checkCompatibility.
-
+#if 0
     if (flags.isKernelEnabled()) {
         if (mKernel.getMatchedKernelRequirements(mat.framework.mKernels, kernelLevel(), error)
                 .empty()) {
             return false;
         }
     }
+#endif
 
     if (flags.isAvbEnabled()) {
         const Version& matAvb = mat.framework.mAvbMetaVersion;
-- 
2.17.1

